compiler/rustc_span/src/span_encoding.rs

```diff
@@ -42,13 +36,11 @@ use rustc_data_structures::fx::FxIndexSet;
 /// - `base` is 32 bits in both `Span` and `SpanData`, which means that `base`
 ///   values never cause interning. The number of bits needed for `base`
 ///   depends on the crate size. 32 bits allows up to 4 GiB of code in a crate.
-///   `script-servo` is the largest crate in `rustc-perf`, requiring 26 bits
-///   for some spans.
```

Comment:
Why did you remove this?

Reply:
This seemed like it might be out of date. I didn't see a `rustc-perf` in `compiler/`. Perhaps I'm missing the meaning entirely here.

Reply:
It's out of tree: https://github.com/rust-lang/rustc-perf

Can you add the comment back?

Reply:
Ah! Will do.